shpak.js

```diff
@@ -5,19 +5,47 @@ const {Save} = require('./src/fileManager')
 const {S, P} = require('./src/constManager')
 
 function Shpak () {
+  const conf = {
+    entry: '',
+    destDir: '',
+    destFile: ''
+  }
   function process () {
     const s_pipe = pipe(loadEntryPoint, parseModules, saveModule)
     return s_pipe()
   }
   function loadEntryPoint () {
+    let entryFile
     try {
-      return S.join(__dirname, require(S.join(__dirname,S.entry())).entry)
-    } catch (err) {
-      //TODO add error object assignation
+      entryFile = require(S.join(__dirname,S.entry()))
+    } catch (e) {
+      throw 'There is no entry file <shpak.config.js>'
     }
+
+    conf.entry = S.join(__dirname, entryFile.entry)
+
+    conf.destDir = entryFile.destDir ? S.join(__dirname, entryFile.destDir) : __dirname
```

Comment:
We don't need that. We can store dest dir in local const instead of shpak function closure.